Tweak/Core/KayokoCore.m

```diff
@@ -48,9 +66,27 @@ static void reload() {
     }
 }
 
+static void pasted() {
+    if (pfPlaySoundEffects) {
+        /* I don’t like these fixed paths, whatever. */
+        static dispatch_once_t onceToken;
+        static SystemSoundID soundID;
+        dispatch_once(&onceToken, ^{
+            OSStatus err = AudioServicesCreateSystemSoundID((__bridge CFURLRef)[NSURL fileURLWithPath:@"/Library/PreferenceBundles/KayokoPreferences.bundle/Paste.aiff"], &soundID);
+            if (err != kAudioServicesNoError) {
+                err = AudioServicesCreateSystemSoundID((__bridge CFURLRef)[NSURL fileURLWithPath:@"/var/jb/Library/PreferenceBundles/KayokoPreferences.bundle/Paste.aiff"], &soundID);
+            }
+        });
+        AudioServicesPlaySystemSound(soundID);
+    }
+    if (pfPlayHapticFeedback) {
+        AudioServicesPlaySystemSound(1519);
+    }
+}
+
 #pragma mark - Preferences
 
-static void load_preferences() {
+static void load_preferences(CFNotificationCenterRef center, void *observer, CFNotificationName name, const void *object, CFDictionaryRef userInfo) {
```

Comment:
Hint: uses these arguments to distinguish between the initial load and the notification callback.